mo

```diff
@@ -30,13 +32,18 @@ mo() (
     local moContent
 
     IFS=$' \n\t'
-    
+
     if [[ $# -gt 0 ]]; then
         case "$1" in
             -h|--h|--he|--hel|--help)
                 moUsage "$0"
                 exit 0
                 ;;
+
+            --source=*)
+                . "${1#--source=}"
```

Comment:
There's no checking to see if `--source=` was passed without a filename, as in ...

    mo --source=